Docs - Improve SEO (#99)
iotaledger/IOTA-2.0-Research-Specifications: docs/6.3FastProbabilisticConsensus.md

---

> description: Probabilistic Consensus (FPC), is triggered if the eligibility of a  message or transaction object is uncertain.

Reviewer: ```suggestion
description: Fast Probabilistic Consensus (FPC) checks if a message or a transaction is eligible when it is not yet certain.
```
